scripts/storage-proxy-spawn-ceph-clusters/install_storage_proxy.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+
+sudo chmod 755 /
```

Comment:
It's not a really good idea to update the permission of root folder. Let's just remove this line.

Reply:
This is for installing into vagrant VM for testing purposes of Ceph. I remember I got some permission issue in vagrant if it wasn't set like this

Reply:
Ah, I was completely misunderstanding context. If this is for installing storage-proxy implementations under Vagrant VM, then all these steps will be understandable. Please ignore my previous reviews on this script.